### ISSUE TITLE
Table 1 and Table 2 test set

### ISSUE THREAD

**Issue by jpyusiong** [mrharicot/monodepth#83]
Hi,

I would like to ask if the test set used in these two tables (Table 1 and Table 2) are the same, the KITTI 2015 stereo 200 training set disparity images? I am wondering where was the EIGEN test set of 697 used?

**Comment by mrharicot** [comment]
Hi,
As we state in the paper, these two tables use different test data.
Table 1: 200 ground truth disparities from the KITTI stereo 2015 benchmark (utils/filenames/kitti_stereo_2015_test_files.txt)
Table 2: 697 reprojected lidar point clouds from the raw KITTI data (following Eigen et al.) (utils/filenames/eigen_test_files.txt)
I hope this helps!

**Comment by jpyusiong** [comment]
Thanks for the clarification.
